src/scala/microsoft-spark-2.3.x/src/main/scala/org/apache/spark/sql/api/dotnet/SQLUtils.scala

```diff
@@ -34,4 +36,21 @@ object SQLUtils {
       broadcastVars,
       accumulator)
   }
+
+  /**
+   * Helper method to create typed MemoryStreams intended for use in unit tests.
+   * @param sqlContext The SQLContext.
+   * @param streamType The type of memory stream to create. This string is the `Name`
+   *                   property of the dotnet type.
+   * @return A typed MemoryStream.
+   */
+  def createMemoryStream(implicit sqlContext: SQLContext, streamType: String): MemoryStream[_] = {
+    import sqlContext.implicits._
+
+    streamType match {
+      case "Int32" => MemoryStream[Int]
+      case "String" => MemoryStream[String]
+      case _ => throw new Exception(s"$streamType not supported")
```

Comment:
Why are `int` and `string` the only types of `MemoryStream` we are supporting?

Reply:
It's only for unit testing.  If anyone feels like they need a different type, they can add support themselves.

Reply:
Got it. Also, nit, but I'm wondering if we want to put this function in this file or a separate `TestUtils` file given how it is meant to be used for unit tests only.

Reply:
Yea, can we move this out to `TestUtils`?

Reply:
Also, we can put it under `org.apache.spark.sql.test`